modules/task/backend.go

```diff
@@ -78,31 +74,14 @@ func (b NopBackend) IsRunning() bool {
 type inMemBackend struct {
 	bufQueue  chan []byte
 	isRunning bool
+	isStopped bool
```

Comment:
Feels weird to have both isRunning and isStopped -- shouldn't they always be the opposite of each other?